raiden-dapp/src/components/AddressInput.vue

```diff
@@ -197,11 +197,13 @@ export default class AddressInput extends Mixins(BlockieMixin) {
   }
 }
 
-.address-input /deep/ .v-text-field__details {
+//noinspection CssInvalidPseudoSelector
```

Comment:
Any way to disable this globally? eslint and friends usually support tweaking individual configs

Reply:
From [vue's docs](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors), it seems both syntax are supported

Reply:
So as soon as I switched from `node-sass` to `sass` as per the vuetify upgrade notes the `/deep/` operator started failing. I tried the `>>>` but it seems to not work at all. The documentation actually says that the `sass` preprocessor might have issues with `>>>`

Reply:
Ok it seems that this error was an IDE error, I removed the comment and reduced the severity to warning.